internal/isoutil/isoutil.go

```diff
@@ -141,43 +139,17 @@ func (h *installerHandler) Create(outPath string, volumeLabel string) error {
 	}
 
 	d.LogicalBlocksize = 2048
-	fspec := disk.FilesystemSpec{Partition: 0, FSType: filesystem.TypeISO9660, VolumeLabel: volumeLabel}
+	fspec := disk.FilesystemSpec{
+		Partition:   0,
+		FSType:      filesystem.TypeISO9660,
+		VolumeLabel: volumeLabel,
+		WorkDir:     h.workDir,
```

Comment:
don't we still need to clean the work dir?

Reply:
Not if we call finalize.

I actually added test cases to ensure the directory was removed.